_includes/footer.html

```diff
@@ -20,53 +20,63 @@
             <div class="row">
                 <div class="col-xs-12 col-sm-3 col-md-3">
                     <ul class="footer_links">
-                        <li><b><a href="https://www.docker.com/why-docker">Why Docker?</a></b></li>
-                        <li><a href="https://www.docker.com/what-container">What is a Container?</a></li>
-                        <li><a href="https://docker.events.cube365.net/dockercon/">DockerCon</a></li>
-                        <li><b><a href="https://www.docker.com/products/overview">Products</a></b></li>
+                        <li><b><a href="https://www.docker.com/products">Product offerings</a></b></li>
+                        <li><a href="https://www.docker.com/products/personal">Docker Personal</a></li>
+                        <li><a href="https://www.docker.com/products/pro">Docker Pro</a></li>
+                        <li><a href="https://www.docker.com/products/team">Docker Team</a></li>
+                        <li><a href="https://www.docker.com/products/business">Docker Business</a></li>
+                        <li><a href="https://www.docker.com/products">Compare Subscriptions</a></li>
+                        <li><b><a href="https://www.docker.com/">Features</a></b></li>
                         <li><a href="https://www.docker.com/products/docker-desktop">Docker Desktop</a></li>
                         <li><a href="https://www.docker.com/products/docker-hub">Docker Hub</a></li>
-                        <li><a href="https://www.docker.com/roadmap">Docker Product Roadmap</a></li>
-                        <li><b><a href="https://www.docker.com/products/docker-desktop">Features</a></b></li>
+                        <li><a href="https://www.docker.com/products/secure-software-supply-chain">Secure Software Supply Chain</a></li>
                         <li><a href="https://www.docker.com/products/container-runtime">Container Runtime</a></li>
                         <li><a href="https://www.docker.com/products/developer-tools">Developer Tools</a></li>
+                        <li><a href="https://www.docker.com/partners">Trusted Content</a></li>
+                        <li><a href="https://www.docker.com/roadmap">Docker Product Roadmap</a></li>
                     </ul>
                 </div>
                 <div class="col-xs-12 col-sm-3 col-md-3">
                     <ul class="footer_links">
                         <li><b><a href="https://www.docker.com/products/docker-desktop">Developers</a></b></li>
                         <li><a href="https://www.docker.com/use-cases">Use Cases</a></li>
+                        <li><a href="https://docs.docker.com/get-started/">Get Started</a></li>
```

Comment:
Feels a bit odd to have a docs.docker.com link in the footer as well 🤔

Reply:
Yea, basically mirrored the footer on WWW to reflect the latest changes. Wanted to point the Get started link to Docs instead of the Get started on WWW